homeassistant/components/skybell/config_flow.py

```diff
@@ -17,6 +18,40 @@
 class SkybellFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Skybell."""
 
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        self._email: str
+
+    async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
+        """Handle a reauthorization flow request."""
+        self._email = entry_data[CONF_EMAIL]
+        return await self.async_step_reauth_confirm()
+
+    async def async_step_reauth_confirm(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Handle user's reauth credentials."""
+        errors = {}
+        if user_input:
+            password = user_input[CONF_PASSWORD]
+            entry_id = self.context["entry_id"]
+            if entry := self.hass.config_entries.async_get_entry(entry_id):
+                _, error = await self._async_validate_input(self._email, password)
+                if error is None:
+                    self.hass.config_entries.async_update_entry(
+                        entry,
+                        data=entry.data | user_input,
+                    )
+                    await self.hass.config_entries.async_reload(entry.entry_id)
+                    return self.async_abort(reason="reauth_successful")
+
+            errors["base"] = error
+        return self.async_show_form(
+            step_id="reauth_confirm",
+            data_schema=vol.Schema({vol.Required(CONF_PASSWORD): str}),
+            errors=errors,
+        )
```

Comment:
Would it make sense to add the username/email as a description placeholder?
https://github.com/home-assistant/core/blob/680a477009f4f1f986a55f76988afb86339897b2/homeassistant/components/renault/config_flow.py#L139-L141
https://github.com/home-assistant/core/blob/680a477009f4f1f986a55f76988afb86339897b2/homeassistant/components/renault/strings.json#L22